guide/src/components/IconSection.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import Scroll from 'react-scroll';
+import R from 'ramda';
+import {  Hr, P, Title, Div, Section } from 'cyverse-ui'; 
+import * as icons from 'cyverse-ui/icons';
+
+import theme from '../theme';
+import { Figure } from './';
+import MarkdownElement from './MarkdownElement';
+import { IconEx } from '../examples';
+import IconExCode from '!raw-loader!../examples/IconEx';
+
+const scroller = Scroll.scroller;
+const ScrollAnchor = Scroll.Element;
+
+const IconSection = React.createClass({
+    IconCell(icon) {
+        const Icon = icon[1];
+        const style = {
+            cell: {
+                display: "flex",
+                flexDirection: "column",
+                alignItems: "center",
+                width: "150px",
+                height: "150px"
+            }
+        }
+        return (
+            <Div
+                key={ icon[0] }
+                style={style.cell}
+            >
+                <Div mb={3}>
+                    <Icon size={ 50 }/>
+                </Div>
+                <P>
+                    { icon[0] }
+                </P>
+            </Div>
+        );
+    },
```

Comment:
No benefit really. I have always just used methods for these things until I have some reason to make it available.  Probably cleaner looking to define it out of createClass as a functional component though.